clients/ctl/admin-ui/src/features/system/ManualSystemFlow.tsx

```diff
@@ -42,20 +45,24 @@ const ConfigureSteps = ({
 
 const ManualSystemFlow = () => {
   const router = useRouter();
+  const dispatch = useAppDispatch();
+
   const [currentStepIndex, setCurrentStepIndex] = useState(0);
-  const [newSystem, setNewSystem] = useState<System | null>(null);
+  const activeSystem = useAppSelector(selectActiveSystem);
 
   const returnToNew = () => {
     router.push("/system/new");
+    dispatch(setActiveSystem(null));
```

Comment:
I expected clicking "Cancel" while editing to take me back to the system list (from whence I came 🎩 ) but of course it took me to the "new" page. Maybe instead of routing directly to "new" it could `router.back()`.

Reply:
good call!